src/main/scala/faces/utils/Helpers.scala

```diff
@@ -91,23 +91,62 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
     new File(bgPath).listFiles.filter(_.getName.endsWith(bgType)).toIndexedSeq
   }
 
-  // center the face around a point between the ears and nose (change here if you want to center another point)
-  def center(rps: RenderParameter): RenderParameter = {
-    val lmLeftEar: TLMSLandmark2D = renderer.renderLandmark("left.eye.pupil.center", rps).get
-    //ear.tragus.tip", rps).get // get is safe, checked during construction
-    val lmRightEar: TLMSLandmark2D = renderer.renderLandmark("right.eye.pupil.center", rps).get
+  // center the face around chin, eyes, nose and ears and crop face box (change here if you want to center another point)
+  def centerFaceBox(rps: RenderParameter, scaling : Double = 1): RenderParameter = {
+    val lmTop: TLMSLandmark2D = renderer.renderLandmark("center.front.trichion", rps).get
+
     val lmNose: TLMSLandmark2D = renderer.renderLandmark("center.nose.tip", rps).get
 
+    val lmChin: TLMSLandmark2D = renderer.renderLandmark("center.chin.tip", rps).get
+
+    val lmLeftEar: TLMSLandmark2D = renderer.renderLandmark("left.ear.helix.outer", rps).get
+    val lmRightEar: TLMSLandmark2D = renderer.renderLandmark("right.ear.helix.outer", rps).get
+
+    val lmLeftEyebrow: TLMSLandmark2D = renderer.renderLandmark("left.eyebrow.bend.lower", rps).get
+    val lmRightEyebrow: TLMSLandmark2D = renderer.renderLandmark("right.eyebrow.bend.lower", rps).get
+
+    val lms = IndexedSeq(lmTop, lmNose, lmChin, lmLeftEar, lmRightEar, lmLeftEyebrow, lmRightEyebrow)
+
+
+    val xmax = lms.maxBy( _.point.x ).point.x
+    val xmin = lms.minBy( _.point.x ).point.x
+
+    val ymax = lms.maxBy( _.point.y ).point.y
+    val ymin = lms.minBy( _.point.y ).point.y
+
+    val xcenter = (xmax+xmin)/2.0
+    val ycenter = (ymax+ymin)/2.0
+
+    val maxSide = math.max(xmax-xmin, ymax-ymin) * scaling
+
+    val middleOfFace = Point(xcenter, ycenter)
 
-    val middleOfFace = (lmLeftEar.point + lmRightEar.point.toVector + lmNose.point.toVector).toVector / 3
     val centerOfImage = Point2D(imageWidth / 2, imageHeight / 2)
-    val shift = centerOfImage - middleOfFace
+
+    val centerShift= {
+      val centerShiftLear = (lmLeftEar.point- lmNose.point)
```

Comment:
You should avoid such inconsistent abbreaviations for variable names (use IDE to reduce typing not shortening of names):

```
val centerShiftLear = ...
// consistent would be
val centerShiftLEar = ...
// more readable is
val centerShiftLeftEar = ...
```